addons/context_actions/CfgContext.hpp

```diff
@@ -299,27 +299,27 @@ class EGVAR(context_menu,actions) {
             icon = QPATHTOF(ui\add_ca.paa);
             priority = 2;
             class 10m {
-                displayName = "10 m";
+                displayName = CSTRING(10m);
```

Comment:
Couldn't have this just used the `FORMAT` macro and only localized the meter (moving it to common stringtable or something)?

Reply:
No, you have to localize the entire string, cannot use a combination. That macro is essentially the `format` command with the suffix `_#` as the number of arguments.

Reply:
So `format [CSTRING(m), 10]` and string `"%1 m` only works in SQF?

Reply:
AFAIK you can only do that in SQF.